### PR TITLE
perfschema: turn off current statement update.

### PR DESCRIPTION
**Pull request by coocood**
Temporary fix for https://github.com/pingcap/tidb/issues/1037

### CODE REVIEW

File: perfschema/statement.go [hunk 1]
```diff
@@ -200,11 +200,12 @@ func (ps *perfSchema) EndStatement(state *StatementState) {
 	log.Debugf("EndStatement: sql %s, connection id %d, type %s", state.sqlText, state.connID, state.stmtType)
 
 	record := state2Record(state)
-	err := ps.updateEventsStmtsCurrent(state.connID, record)
-	if err != nil {
-		log.Error("Unable to update events_statements_current table")
-	}
-	err = ps.appendEventsStmtsHistory(record)
+	// TODO: add back when custimized perfomance schema implemented.
```
Comment (qiuyesuifeng) [inline perfschema/statement.go]: custimized -> customized ?

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
@qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**